packages/eslint-config/package.json

```
@@ -15,14 +15,16 @@
   ],
   "dependencies": {
     "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-datarockets-base": "latest"
+    "eslint-config-datarockets-base": "https://gitpkg.now.sh/datarockets/style-guide/packages/eslint-config-base?upgrade-dependencies"
```

```suggestion
    "eslint-config-datarockets-base": "latest"
```

I'm not sure how to tell the old Lerna to use the workspace package instead of pinned version but I don't think we should use `https://gitpkg.now.sh/datarockets/style-guide/packages/eslint-config-base?upgrade-dependencies` in a published version.